aidants_connect_web/templates/aidants_connect_web/id_provider/fi_select_demarche.html

```diff
@@ -11,8 +11,11 @@
 <section class="section section-grey">
   <div class="container">
     <form method="post">
-      <h2 id="welcome_aidant">Sélectionnez le type de démarche que vous allez effectuer</h2>
-      <p id="instructions">En selectionnant une démarche, vous allez FranceConnecter [Nom de l'usager] (Changer d'usager)</p>
+      <h2>Sélectionnez le type de démarche que vous allez effectuer à la place de {{ usager.given_name }} {{ usager.family_name }}</h2>
```

Comment:
Pinaillage encore, mais que penseriez-vous de _au nom de_ plutôt que _à la place de_ ?
J'ai l'impression que cela convoierait davantage la notion de _délégation_ qui est au coeur du service :)

```suggestion
      <h2>Sélectionnez le type de démarche que vous allez effectuer au nom de {{ usager.given_name }} {{ usager.family_name }}</h2>
```

Reply:
comme discuté, on reste sur à la place de :)